client/webserver/site/src/css/main.scss

```diff
@@ -427,10 +427,9 @@ hr.dashed {
   background-color: #7775;
 }
 
-div.form-closer,
-div.corner-bttn {
+div.form-closer {
   position: absolute;
-  right: -15px;
```

Comment:
Before

![image](https://user-images.githubusercontent.com/9373513/181128653-ff6aae14-7d15-47b9-b144-4c3326c8d516.png)

After

![image](https://user-images.githubusercontent.com/9373513/181128713-f11bf31e-4108-4213-823c-d1de644222b2.png)

After, hovering mouse over the X

![image](https://user-images.githubusercontent.com/9373513/181128750-16f96fe9-e38c-4b3b-b193-4c062d6c5035.png)

I can live with that.  But I wonder if we need to reserve space for the X _above_ the rest of the form elements, or have it over top anything else in the form with nice styling.  Right now it is firmly outside the form.